Example/iOS/Views/AnimatedButtonRow.swift

```diff
@@ -57,33 +57,31 @@ final class AnimatedButtonRow: UIView, EpoxyableView {
     guard let content = content else { return }
 
     group.setItems {
-      if let animationName = content.animationName {
```

Comment:
Weird! I wonder how this compiled before considering `content.animationName` is non-optional. Must have been a compiler bug.

Reply:
Filed an issue for this on the Swift repo since I think this source break was probably unintentional: https://github.com/apple/swift/issues/63808